lib/chef-dk/skeletons/code_generator/templates/default/kitchen.yml.erb

```diff
@@ -7,7 +7,6 @@ provisioner:
 
 verifier:
   name: inspec
-  format: doc
```

Comment:
Is there a reason we're turning off the doc format?  This gave us more verbose output that was more useful to users and something people who migrated from Serverspec requested